src/main/scala/com/adobe/api/platform/runtime/metrics/KamonConsumer.scala

```diff
@@ -59,8 +59,46 @@ object KamonConsumer {
       .collect { case e if e.eventType == Activation.typeName => e } //Look for only Activations
       .foreach { e =>
         val a = e.body.asInstanceOf[Activation]
-        Kamon.histogram("waitTime", MeasurementUnit.time.milliseconds).refine("name" -> a.name).record(a.waitTime)
-        Kamon.counter("activations").refine("name" -> a.name).increment()
+        val (namespace, action) = getNamespaceAction(a.name)
+
+        val tags = List(
+          ("namespace" -> namespace),
+          ("action" -> action),
+          ("kind" -> a.kind),
+          ("memory" -> a.memory.toString),
+          ("status" -> a.statusCode.toString))
+
+        Kamon.counter("openwhisk.counter.container.activations").refine(tags: _*).increment()
+        Kamon
+          .counter("openwhisk.counter.container.coldStarts")
+          .refine(tags: _*)
```

Comment:
Instead of use a `List` type for `tags` you can make it a `Map` then you would not need to spread `_*` the arguments